apollo-runtime/src/main/java/com/apollographql/apollo/subscription/OperationMessageSerializer.kt

```diff
@@ -0,0 +1,13 @@
+package com.apollographql.apollo.subscription
+
+import okio.BufferedSink
+import okio.BufferedSource
+import java.io.IOException
+
+interface OperationMessageSerializer {
+  @Throws(IOException::class)
+  fun OperationClientMessage.writeTo(sink: BufferedSink)
```

Comment:
What about `writeClientMessage(sink: BufferedSink, message: OperationClientMessage)` so that it's more symmetrical with `readServerMessage` ? I don't have a strong opinion there so either work for me though.

Reply:
Me neither. I figured the receiver pattern worked well because it would be like a function on `OperationClientMessage` so the implementation looks cleaner.